src/windows/views/zoom_slider.py

```diff
@@ -0,0 +1,508 @@
+                    # SHIFT key pressed (move both handles)
+                    new_left_pos = self.scrollbar_position[0] + delta
+                else:
+                    new_left_pos = self.scrollbar_position[0]
+                new_right_pos = self.scrollbar_position[1] - delta
+
+                if new_left_pos < 0.0:
+                    new_left_pos = 0.0
+                    new_right_pos = self.scroll_bar_rect.width() / self.width()
+                elif new_right_pos > 1.0:
+                    new_left_pos = 1.0 - (self.scroll_bar_rect.width() / self.width())
+                    new_right_pos = 1.0
+
+                self.scrollbar_position = [new_left_pos,
+                                           new_right_pos,
+                                           self.scrollbar_position[2],
+                                           self.scrollbar_position[3]]
+                self.delayed_resize_timer.start()
+
+            elif self.scroll_bar_dragging:
+                # Update scrollbar position
+                delta = (self.mouse_position - mouse_pos) / self.width()
+                new_left_pos = self.scrollbar_position[0] - delta
+                new_right_pos = self.scrollbar_position[1] - delta
+
+                if new_left_pos < 0.0:
+                    new_left_pos = 0.0
+                    new_right_pos = self.scroll_bar_rect.width() / self.width()
+                elif new_right_pos > 1.0:
+                    new_left_pos = 1.0 - (self.scroll_bar_rect.width() / self.width())
+                    new_right_pos = 1.0
+
+                self.scrollbar_position = [new_left_pos,
+                                           new_right_pos,
+                                           self.scrollbar_position[2],
+                                           self.scrollbar_position[3]]
+
+                # Emit signal to scroll Timeline
+                get_app().window.TimelineScroll.emit(self.scrollbar_position[0])
+
+            # Force re-paint
+            self.update()
+
+        # Update mouse position
+        self.mouse_position = mouse_pos
+
+    def resizeEvent(self, event):
+        """Widget resize event"""
+        event.accept()
+        self.delayed_size = self.size()
+        self.delayed_resize_timer.start()
+
+    def delayed_resize_callback(self):
+        """Callback for resize event timer (to delay the resize event, and prevent lots of similar resize events)"""
+        # Get max width of timeline
+        project_duration = get_app().project.get("duration")
+        normalized_scroll_width = self.scrollbar_position[1] - self.scrollbar_position[0]
+        scroll_width_seconds = normalized_scroll_width * project_duration
+        tick_pixels = 100
+        if self.scrollbar_position[3] > 0.0:
+            # Calculate the new zoom factor, based on 100 pixels per tick
+            zoom_factor = scroll_width_seconds / (self.scrollbar_position[3] / tick_pixels)
```

Comment:
The calculation **here** isn't based on 100 pixels per tick, it's based on `tick_pixels`. If the value of `tick_pixels` is changed, that comment becomes a lie, so it shouldn't specify a value that could change.

Reply:
(I will pay you **#ALL_THE_MONEY** if you can find me someone who'll claim that they'll change the comment when they change `tick_pixels`, and not be lying.)

Reply:
I like those odds! haha